st2common/st2common/util/config_loader.py

```diff
@@ -35,6 +35,8 @@
 
 LOG = logging.getLogger(__name__)
 
+DEFAULT_VALUE_MAGIC_MARKER = '~~***---%%--%%--%--%%%***~~~+++'
```

Comment:
I was actually just working to use a more robust approach - ef8b87a5666edca38d928e242cd20f959338f21d.

Will push more changes soon.